suite-common/suite-utils/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@suite-common/suite-utils",
+    "version": "1.0.0",
+    "private": true,
+    "license": "See LICENSE.md in repo root",
+    "sideEffects": false,
+    "main": "src/index",
+    "type": "module",
```

Comment:
How are you creating packages? `type: module` should be already removed from package generator, because it was mistake to use and it broke HMR for desktop suite. Please remove it from all created packages.

Also you will need to update all `jest.config.js` to use `module.exports` instead of `export default`